_security/access-control/permissions.md

```diff
@@ -134,37 +172,117 @@ These permissions are for the cluster and can't be applied granularly. For examp
+
+See [Snapshot APIs]({{site.url}}{{site.baseurl}}/api-reference/snapshots/index/).
+
 - cluster:admin/repository/delete
 - cluster:admin/repository/get
 - cluster:admin/repository/put
 - cluster:admin/repository/verify
+
+### Reroute
+
+See [Cluster manager task throttling]({{site.url}}{{site.baseurl}}/tuning-your-cluster/cluster-manager-task-throttling/).
+
 - cluster:admin/reroute
+
+### Script permissions
+
+See [Script APIs]({{site.url}}{{site.baseurl}}/api-reference/script-apis/index/).
+
 - cluster:admin/script/delete
 - cluster:admin/script/get
 - cluster:admin/script/put
+
+### Update settings permission
+
+See [Update settings]({{site.url}}{{site.baseurl}}api-reference/index-apis/update-settings/) in Index APIs.
+
 - cluster:admin/settings/update
+
+### Snapshot permissions
+
+See [Snapshot APIs]({{site.url}}{{site.baseurl}}/api-reference/snapshots/index/).
+
 - cluster:admin/snapshot/create
 - cluster:admin/snapshot/delete
 - cluster:admin/snapshot/get
 - cluster:admin/snapshot/restore
 - cluster:admin/snapshot/status
 - cluster:admin/snapshot/status*
+
+### Task permissions
+
+See [Tasks]({{site.url}}{{site.baseurl}}/api-reference/tasks/) APIs.
+
 - cluster:admin/tasks/cancel
 - cluster:admin/tasks/test
 - cluster:admin/tasks/testunblock
+
+### Security Analytics permissions
+
+See [API tools]({{site.url}}{{site.baseurl}}/security-analytics/api-tools/index/).
+
+**Permission** | **Description**
+:--- | :---
+cluster:admin/opensearch/securityanalytics/alerts/get | permission to get alerts
+cluster:admin/opensearch/securityanalytics/alerts/ack | permission to acknowledge alerts
+cluster:admin/opensearch/securityanalytics/detector/get | permission to get detectors
+cluster:admin/opensearch/securityanalytics/detector/search | permission to search detectors
+cluster:admin/opensearch/securityanalytics/detector/write | permission to create and update detectors
+cluster:admin/opensearch/securityanalytics/detector/delete | permission to delete detectors
+cluster:admin/opensearch/securityanalytics/findings/get | permission to get findings
+cluster:admin/opensearch/securityanalytics/mapping/get | permission to get field mappings by index
+cluster:admin/opensearch/securityanalytics/mapping/view/get | permission to get field mappings by index and view mapped and unmapped fields
+cluster:admin/opensearch/securityanalytics/mapping/create | permission to create field mappings
+cluster:admin/opensearch/securityanalytics/mapping/update | permission to update field mappings
+cluster:admin/opensearch/securityanalytics/rules/categories | permission to get all rule categories
+cluster:admin/opensearch/securityanalytics/rule/write | permission to create and update rules
+cluster:admin/opensearch/securityanalytics/rule/search | permission to search for rules
+cluster:admin/opensearch/securityanalytics/rules/validate | permission to validate rules
+cluster:admin/opensearch/securityanalytics/rule/delete | permission to delete rules
```

Comment:
Need to add pipe "|" characters to the beginning and end of each line for compatibility (see https://www.markdownguide.org/extended-syntax/#tables for guidance). It technically renders fine this way, but if we change the underlying markdown interpreter then this table could break. Copy-paste this if you want:

```
| **Permission** | **Description** |
| :--- | :--- |
| cluster:admin/opensearch/securityanalytics/alerts/get | permission to get alerts |
| cluster:admin/opensearch/securityanalytics/alerts/ack | permission to acknowledge alerts |
| cluster:admin/opensearch/securityanalytics/detector/get | permission to get detectors |
| cluster:admin/opensearch/securityanalytics/detector/search | permission to search detectors |
| cluster:admin/opensearch/securityanalytics/detector/write | permission to create and update detectors |
| cluster:admin/opensearch/securityanalytics/detector/delete | permission to delete detectors |
| cluster:admin/opensearch/securityanalytics/findings/get | permission to get findings |
| cluster:admin/opensearch/securityanalytics/mapping/get | permission to get field mappings by index |
| cluster:admin/opensearch/securityanalytics/mapping/view/get | permission to get field mappings by index and view mapped and unmapped fields |
| cluster:admin/opensearch/securityanalytics/mapping/create | permission to create field mappings |
| cluster:admin/opensearch/securityanalytics/mapping/update | permission to update field mappings |
| cluster:admin/opensearch/securityanalytics/rules/categories | permission to get all rule categories |
| cluster:admin/opensearch/securityanalytics/rule/write | permission to create and update rules |
| cluster:admin/opensearch/securityanalytics/rule/search | permission to search for rules |
| cluster:admin/opensearch/securityanalytics/rules/validate | permission to validate rules |
| cluster:admin/opensearch/securityanalytics/rule/delete | permission to delete rules |
```